libgin/datacite.go

```diff
@@ -79,7 +79,7 @@ type DataCite struct {
 	Subjects []string `xml:"subjects>subject,omitempty"`
 	// RelatedIdentifiers: References
 	RelatedIdentifiers []RelatedIdentifier `xml:"relatedIdentifiers,omitempty>relatedIdentifier"`
```

Comment:
What is the difference of the `omitempty` between RelatedIdentifier and FundingReferences, why move one and not the other?

Reply:
Not sure.  I remember the issue with FundingReferences was that it caused the list of Funders to break into multiple lists, so maybe it happens with RelatedIdentifiers too and I missed it.  Will get back to you in a bit.

Reply:
Yup.  Same issue.  Fixed now.